ctapipe/conftest.py

```diff
@@ -443,14 +443,41 @@ def dl1_muon_file(dl1_tmp_path):
             f"--input={infile}",
             f"--output={output}",
             "--write-images",
-            "--DataWriter.write_parameters=False",
-            "--DataWriter.Contact.name=αℓℓ the äüöß",
+            "--no-write-parameters",
             "--SimTelEventSource.focal_length_choice=EQUIVALENT",
         ]
         assert run_tool(ProcessorTool(), argv=argv, cwd=dl1_tmp_path) == 0
         return output
 
 
+@pytest.fixture(scope="session")
+def dl1_muon_output_file(dl1_tmp_path, dl1_muon_file):
+    """
+    DL1 file containing images, parameters and muon ring parameters
```

Comment:
This file contains no images and no parameters.  If a file with everything is needed, we don't need `dl1_muon_output_file`, but can use `dl1_muon_file` and add `--write-muon-parameters` there?

Reply:
To clearify the existence of these files, `dl1_muon_file` is a file created from muon simulations containing only images and is used to test the muon-analysis specifically in `ctapipe/image/muon/tests/test_processor.py`. `dl1_muon_output_file` on the other hand is a file containing analysed muon events (therefore e.g. muon efficiencies) used to test e.g. the `HDF5EventSource` for correct handling of the muon nodes.

Reply:
thanks for the clarification